cmake/modules/hunter_windows_to_snf_filename.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015 Sumedh Ghaisas, Ruslan Baratov
+# All rights reserved.
+
+# CMAKE function to convert windows log filename to 8.3 filename
+# For reference check http://stackoverflow.com/a/10227715/2288008
+function(hunter_windows_to_snf_filename path new_path_variable)
+  set(conversion_cmd
+    "cmd" 
+    "/c" 
+    "for" 
+    "%A" 
+    "in" 
+    ("${path}") 
+    "do" 
+    "@echo"
+    "%~sA"
+	)
+  
+  execute_process(
```

Comment:
Command will fail if system is not Windows. Please add check:
```cmake
if(NOT CMAKE_HOST_WIN32)
  hunter_internal_error("Expecting Windows platform")
endif()
```

See https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_WIN32.html